src/lib/utilities/handle-error.ts

```diff
@@ -12,12 +13,12 @@ export const handleError = (
   isBrowser = browser,
 ): void => {
   if (isUnauthorized(error) && isBrowser) {
-    window.location.assign(routeForLoginPage());
+    window.location.assign(routeForLoginPage(error?.['message']));
```

Comment:
Why not `error?.message`?

Reply:
changed the error type back to `any` so it allows to use `error?.message`